syncano/models/manager.py

```diff
@@ -828,6 +828,26 @@ def _get_endpoint_properties(self):
         return self.model._meta.resolve_endpoint(self.endpoint, defaults), defaults
 
 
+class RuntimeChoices(object):
```

Comment:
As I understood, you wanted to implement something that reasembles metaenum, but your solution doesn't work like this, because you set attributes on class instance, not on a class itself.